mizar/obj/net.py

```diff
@@ -120,6 +126,12 @@ def set_status(self, status):
     def get_gw_ip(self):
         return str(self.cidr.get_ip(1))
 
+    def set_remote_deployed(self, remote_deployed):
```

Comment:
Can we make this parameter default false?

Reply:
Added